.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,94 @@
+name: Unit Tests
+
+on: pull_request
+
+jobs:
+  tests:
+    name: "WP: ${{ matrix.wp_version }} - PHP: ${{ matrix.php }} - ES: ${{ matrix.es_version }} (MU: ${{ matrix.multisite }})"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false # do not fail fast, let all the failing tests fail.
+      matrix:
+        php: [8.0]
+        es_version: [7.17.5]
+        multisite: [0]
+        wp_version: ["latest"]
```

Comment:
This is confusing - in our conversation you said the PHP 8 check isn't being run? Is there a way to specify all items in the matrix on the same "level" vs. having some defined here and some in the `include` section?

Reply:
It is being run, ES: 8.3.2 is the one which is not running since there is no support for it yet.